Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -173,6 +217,44 @@ def urlscan_submit_url(client):
     return r
 
 
+def create_relationship(scan_type, field, entity_a, object_type_a, entity_b, object_type_b):
+    relations_type = 'indicatorToIndicator'
+    entity_a_family = 'Indicator'
+    entity_b_family = 'Indicator'
+    brand = 'urlscan.io'
+    return EntityRelation(name=RELATIONSHIP_TYPE.get(scan_type, {}).get(field, {}).get('name', ''),
```

Comment:
we should have a simple wrapper in common server for this that puts `indicatorToIndicator`, and `Indicator` in the entity_a_family/entity_b_family by default.